### PR TITLE
Feature/us19575/groups on site button

### PR DESCRIPTION
**Pull request by kellgauthier**
## Problem
Change button color for onsite group details page to match rebrand criteria (orange button/ blue text) 

[Rally](https://rally1.rallydev.com/#/66096747656ud/detail/userstory/393973364824?fdp=true)

## Solution
Created orange button with blue text and implemented style to on-site groups details page. 

## Testing
Run crds-net locally, and navigate to any on-site groups details page. 
Example url - /groups/onsite/COME-AS-YOU-ARE-FLORENCE-2019/columbus/
Screen Shot 2020-05-22 at 11 00 43 AM

### CODE REVIEW

File: _includes/groups/_meeting-details.html [hunk 1]
```diff
@@ -14,5 +14,5 @@
   {% endif %}
 </p>
 {% if meeting.registration_link %}
-  <p class="push-top"><a href="{{ meeting.registration_link }}" class="btn btn-cyan" type="button">Register</a></p>
+  <p class="push-top"><a href="{{ meeting.registration_link }}" class="btn btn-orange-blue" type="button">Register</a></p>
```
Comment (tcmacdonald) [inline _includes/groups/_meeting-details.html:17]: Where is `btn-orange-blue` defined?

Comment (kellgauthier) [reply _includes/groups/_meeting-details.html:17]: Resolved on crds-styles. 
[CRDS-STYLES](https://github.com/crdschurch/crds-styles/pull/429)

Comment (kellgauthier) [reply _includes/groups/_meeting-details.html:17]: https://github.com/crdschurch/crds-styles/pull/432

**Comment by silasbaur** [comment]
@kellgauthier Why was `locations.scss` deleted. Also your build is failing

**Comment by silasbaur** [comment]
@kellgauthier Updates in crds-styles should be done in the crds-styles repo

**Pull request closed without merging**